buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1953,8 +1953,10 @@ class BeamModulePlugin implements Plugin<Project> {
       }
       project.ext.addPortableWordCountTasks = {
         ->
-        addPortableWordCountTask(false)
-        addPortableWordCountTask(true)
+        addPortableWordCountTask(false, "PortableRunner")
```

Comment:
Is there anything that we are testing with PortableRunner that isn't included in FlinkRunner? If not, perhaps just replace those?

Side question for follow-up: Since these tasks are Flink specific, should they be renamed and possibly moved out of BeamModulePlugin?

Reply:
Testing PortableRunner independently is important. Theoretically, the PortableRunner is a subset of FlinkRunner, but there are still things that can go wrong with the PortableRunner when it runs without the orchestration support of the FlinkRunner.

Reply:
> Theoretically, the PortableRunner is a subset of FlinkRunner, but there are still things that can go wrong

+1 As long as PortableRunner remains a publicly documented option (and it will probably have to for at least a little while longer) we should test it. Also, they are testing slightly different things -- PortableRunner starts a containerized job server, while FlinkRunner (when local) starts the job server in a Java subprocess.

> Since these tasks are Flink specific, should they be renamed and possibly moved out of BeamModulePlugin?

I'm going to add these tasks to Spark (eventually).